loops/lang/en-EN/game.xml

```diff
@@ -1762,7 +1762,7 @@
         <restoration>
             <label>Restoration</label>
             <tooltip><![CDATA[
-            You've gone all this way without learning how to heal people, or even yourself. You should probably get around to learning that. Mana cost is reduced by Wizard College Grade.
+            You've gone all this way without learning how to heal people, or even yourself. You should probably get around to learning that.<br>Mana cost is reduced by Wizard College Grade.
```

Comment:
This sounds weird, because we literally heal the sick every loop in zone 1.

Maybe we could word it as healing people with magic? It should probably also give some kind of bonus to heal the sick.

```suggestion
            You've gone all this way without learning how to use your magic to heal people, or even yourself. You should probably get around to learning that.<br>Mana cost is reduced by Wizard College Grade.
```